doc/apiv3-documentation.apib

```diff
@@ -334,14 +334,14 @@ the amount of elements returned in this response and action links to retrieve th
 | total    | The total amount of elements available in the collection        | Integer | always                      |
 | pageSize | Amount of elements that a response will hold                    | Integer | when paginated              |
 | count    | Actual amount of elements in this response                      | Integer | always                      |
-| offset   | Amount of elements preceding the first element of this response | Integer | when offset based available |
+| offset   | The page number that is requested from paginated collection     | Integer | when offset based available |
 
 ## Links
 
 | Link             | Description                                                              | Availability                |
 |:----------------:| ------------------------------------------------------------------------ | --------------------------- |
 | self             | Link to the current page in the collection                               | always                      |
-| changeSize       | Templated link to change the page size, preserving the relative position | when paginated              |
+| changeSize       | Templated link to change the page size, might change relative position   | when paginated              |
```

Comment:
I should have never written that the position is preserved, since this carries assumptions about the direction of traversal.